packages/components/src/internal/query/filter.ts

```diff
@@ -233,9 +368,21 @@ export function getLabKeySql(filter: Filter.IFilter, jsonType: JsonType): string
             getLabKeySqlValue(filter.getValue(), jsonType) +
             ')'
         );
+    } else if (filterType.getURLSuffix() === Filter.Types.CONTAINS.getURLSuffix()) {
+        return getContainsFullClause(filter, jsonType);
+    } else if (filterType.getURLSuffix() === Filter.Types.DOES_NOT_CONTAIN.getURLSuffix()) {
+        return getNotContainsFullClause(filter, jsonType);
+    } else if (filterType.getURLSuffix() === Filter.Types.STARTS_WITH.getURLSuffix()) {
+        return getNotContainsFullClause(filter, jsonType);
```

Comment:
something seems off here? maybe it is a copy-paste issue, but there is an elseif for `DOES_NOT_CONTAIN` and this `STARTS_WITH` which have the same return value and then it is followed by another `STARTS_WITH` that has a different return value. I think lines 375 and 376 just need to be removed, right?